docs/guide/templates.md

```diff
@@ -120,15 +120,42 @@ end
 Since 2.55.0
 {: .label }
 
-To render a parent component's template from a subclass, call `render_parent`:
+To render a parent component's template from a subclass' template, use `#render_parent`:
 
 ```erb
 <%# my_link_component.html.erb %>
 <div class="base-component-template">
-  <% render_parent %>
+  <%= render_parent %>
 </div>
 ```
 
+If the parent supports the current variant, the variant will automatically be rendered.
+
+`#render_parent` also works with inline templates:
+
+```ruby
+class MyComponent < ViewComponent::Base
+  erb_template <<~ERB
+    <div>
+      <%= render_parent %>
+    </div>
+  ERB
+end
+```
+
+To render a parent component's template from a `#call` method, call `super`.
```

Comment:
Is this still true? 

Is it worth updating this, or removing this section since `render_parent` now works with all template definition approaches?

Reply:
Hmm good question! I didn't write tests for this case, so I can't say if it'll work or not. Let me look into it.

Reply:
Ok, so I wrote tests for this case and confirmed something I was anxious about. `#render_parent` works fine if you use `#content_tag` (or `#capture` directly), eg:

```ruby
def call
  content_tag("div") do
    render_parent
  end
end
```

But because `#render_parent` always returns `nil` to avoid double-rendering, this doesn't work:

```ruby
def call
  "<div>#{render_parent}</div>"
end
```

That's not a huge problem, but something I will definitely mention in the docs.

Reply:
I ended up adding a new method, `#render_parent_to_string` to help with the inline template case.

Reply:
I know we wanted to avoid it, but isn't this what capture does for us?

Reply:
Should we add `render_parent_to_string` vs adding capture here? IMO even with the performance penalty I think it would make sense to keep our API surface smaller and make the expected method work as-is.

Reply:
It turns out my test components were inheriting from the wrong base and the tests therefore passed without actually testing what I wanted. I merged before realizing. Long story short: we don't actually need the `capture`.